lib/Models/Catalog/Ows/WebFeatureServiceCatalogItem.ts

```diff
@@ -341,15 +369,15 @@ class WebFeatureServiceCatalogItem extends GetCapabilitiesMixin(
     };
 
     const supportsGeojson =
-      hasOutputFormat(getCapabilitiesStratum.capabilities.outputTypes) ||
+      hasJsonOutputFormat(getCapabilitiesStratum.capabilities.outputTypes) ||
       [
         ...getCapabilitiesStratum.capabilitiesFeatureTypes.values()
       ].reduce<boolean>(
         (hasGeojson, current) =>
-          hasGeojson && hasOutputFormat(current?.OutputFormats),
+          hasGeojson && hasJsonOutputFormat(current?.OutputFormats),
         true
       );
-
+    ``;
```

Comment:
I think all of this logic should be in your new `@computed get outputFormat() ...` in `GetCapabilitiesStratum`

Then you could just have `outputFormat: this.outputFormat` (see next comment).

This means that the outputFormat can be overridden in a JSON catalog file if needed.

Reply:
Done